controllers/admin/AdminTranslationsController.php

```diff
@@ -3047,8 +3053,11 @@ protected function getAllModuleFiles($modules, $root_dir = null, $lang, $is_defa
      */
     public function initFormModules()
     {
-        // Get list of modules
-        $modules = $this->getListModules();
+        // Get list of installed modules
+        $installed_modules = $this->getListModules();
+
+        // get selected module
+        $modules[0] = Tools::getValue('module');
```

Comment:
if you don't want to have all modules, maybe it would be better to change it to `$module = Tools::getValue('module')` and change this empty check below? cleaner code :+1: